2018-09-27 15:00:41 [ ERROR ] [ DefaultSingletonBeanRegistry.java :581(destroyBean)] Destroy method on bean with name 'org.springframework.context.annotation.internalScheduledAnnotationProcessor' threw an exception
java.lang.IllegalStateException: ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: Root WebApplicationContext: startup date [Thu Sep 27 15:00:20 CST 2018]; root of context hierarchy
	at org.springframework.context.support.AbstractApplicationContext.getApplicationEventMulticaster(AbstractApplicationContext.java:404)
	at org.springframework.context.support.ApplicationListenerDetector.postProcessBeforeDestruction(ApplicationListenerDetector.java:97)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:253)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:578)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:554)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:523)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1033)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:555)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:800)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:444)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:294)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1349)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.startWebapp(JettyWebAppContext.java:296)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1342)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:741)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:505)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:365)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.doStart(ContextHandlerCollection.java:163)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.server.Server.start(Server.java:387)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.server.Server.doStart(Server.java:354)
	at org.eclipse.jetty.maven.plugin.JettyServer.doStart(JettyServer.java:73)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.maven.plugin.AbstractJettyMojo.startJetty(AbstractJettyMojo.java:534)
	at org.eclipse.jetty.maven.plugin.AbstractJettyMojo.execute(AbstractJettyMojo.java:357)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo.execute(JettyRunMojo.java:167)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
2018-09-27 15:00:41 [ ERROR ] [ ContextLoader.java :350(initWebApplicationContext)] Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'shiroFilter' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'securityManager' while setting bean property 'securityManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'securityManager' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'upmsRealm' while setting bean property 'realms' with key [0]; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is org.I0Itec.zkclient.exception.ZkException: Unable to connect to zkserver:2181
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.registerBeanPostProcessors(PostProcessorRegistrationDelegate.java:235)
	at org.springframework.context.support.AbstractApplicationContext.registerBeanPostProcessors(AbstractApplicationContext.java:702)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:527)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:800)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:444)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:294)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1349)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.startWebapp(JettyWebAppContext.java:296)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1342)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:741)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:505)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:365)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.doStart(ContextHandlerCollection.java:163)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:132)
	at org.eclipse.jetty.server.Server.start(Server.java:387)
	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:114)
	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:61)
	at org.eclipse.jetty.server.Server.doStart(Server.java:354)
	at org.eclipse.jetty.maven.plugin.JettyServer.doStart(JettyServer.java:73)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.maven.plugin.AbstractJettyMojo.startJetty(AbstractJettyMojo.java:534)
	at org.eclipse.jetty.maven.plugin.AbstractJettyMojo.execute(AbstractJettyMojo.java:357)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo.execute(JettyRunMojo.java:167)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
	at org.codehaus.classworlds.Launcher.main(Launcher.java:47)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'securityManager' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'upmsRealm' while setting bean property 'realms' with key [0]; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is org.I0Itec.zkclient.exception.ZkException: Unable to connect to zkserver:2181
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedList(BeanDefinitionValueResolver.java:382)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:157)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 68 more
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is org.I0Itec.zkclient.exception.ZkException: Unable to connect to zkserver:2181
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:588)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:366)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1264)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 80 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is org.I0Itec.zkclient.exception.ZkException: Unable to connect to zkserver:2181
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:175)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1634)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:254)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.addCandidateEntry(DefaultListableBeanFactory.java:1309)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1275)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1101)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1066)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:585)
	... 90 more
Caused by: org.I0Itec.zkclient.exception.ZkException: Unable to connect to zkserver:2181
	at org.I0Itec.zkclient.ZkConnection.connect(ZkConnection.java:66)
	at org.I0Itec.zkclient.ZkClient.connect(ZkClient.java:872)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:98)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:92)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:76)
	at org.I0Itec.zkclient.ZkClient.<init>(ZkClient.java:72)
	at com.alibaba.dubbo.remoting.zookeeper.zkclient.ZkclientZookeeperClient.<init>(ZkclientZookeeperClient.java:25)
	at com.alibaba.dubbo.remoting.zookeeper.zkclient.ZkclientZookeeperTransporter.connect(ZkclientZookeeperTransporter.java:10)
	at com.alibaba.dubbo.remoting.zookeeper.ZookeeperTransporter$Adaptive.connect(ZookeeperTransporter$Adaptive.java)
	at com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistry.<init>(ZookeeperRegistry.java:69)
	at com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistryFactory.createRegistry(ZookeeperRegistryFactory.java:37)
	at com.alibaba.dubbo.registry.support.AbstractRegistryFactory.getRegistry(AbstractRegistryFactory.java:95)
	at com.alibaba.dubbo.registry.RegistryFactory$Adaptive.getRegistry(RegistryFactory$Adaptive.java)
	at com.alibaba.dubbo.registry.integration.RegistryProtocol.refer(RegistryProtocol.java:254)
	at com.alibaba.dubbo.rpc.protocol.ProtocolListenerWrapper.refer(ProtocolListenerWrapper.java:63)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper.refer(ProtocolFilterWrapper.java:99)
	at com.alibaba.dubbo.rpc.Protocol$Adaptive.refer(Protocol$Adaptive.java)
	at com.alibaba.dubbo.config.ReferenceConfig.createProxy(ReferenceConfig.java:379)
	at com.alibaba.dubbo.config.ReferenceConfig.init(ReferenceConfig.java:320)
	at com.alibaba.dubbo.config.ReferenceConfig.get(ReferenceConfig.java:159)
	at com.alibaba.dubbo.config.spring.ReferenceBean.getObject(ReferenceBean.java:65)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:168)
	... 100 more
Caused by: java.net.UnknownHostException: zkserver
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.zookeeper.ClientCnxn.<init>(ClientCnxn.java:386)
	at org.apache.zookeeper.ClientCnxn.<init>(ClientCnxn.java:331)
	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:377)
	at org.I0Itec.zkclient.ZkConnection.connect(ZkConnection.java:64)
	... 121 more
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:06:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:06:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:06:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:11:27 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:13:22 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:15:13 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:17:04 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:01 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:24:02 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:26:35 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:26:36 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:28:34 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:31:24 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:31:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:34:03 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:36:56 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:10 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:42:29 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:28 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:42 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:42 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:42 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:57 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:47:58 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:44 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 15:59:45 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:02:42 [ ERROR ] [ ContextLoader.java :350(initWebApplicationContext)] Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'org.springframework.cache.interceptor.CacheInterceptor#0': Cannot resolve reference to bean 'cacheManager' while setting bean property 'cacheManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'cacheManager' defined in file [F:\codebase\code\tour-tmsp\tmsp-server\target\classes\applicationContext-ehcache.xml]: Cannot resolve reference to bean 'cacheManagerFactory' while setting bean property 'cacheManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'cacheManagerFactory' defined in file [F:\codebase\code\tour-tmsp\tmsp-server\target\classes\applicationContext-ehcache.xml]: Invocation of init method failed; nested exception is net.sf.ehcache.CacheException: Failed to parse EhCache configuration resource
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:761)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:866)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:800)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:444)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:294)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1349)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.startWebapp(JettyWebAppContext.java:296)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1342)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:741)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:505)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:365)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo.restartWebApp(JettyRunMojo.java:541)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo$1.filesChanged(JettyRunMojo.java:494)
	at org.eclipse.jetty.util.Scanner.reportBulkChanges(Scanner.java:685)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:539)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:391)
	at org.eclipse.jetty.util.Scanner$1.run(Scanner.java:329)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'cacheManager' defined in file [F:\codebase\code\tour-tmsp\tmsp-server\target\classes\applicationContext-ehcache.xml]: Cannot resolve reference to bean 'cacheManagerFactory' while setting bean property 'cacheManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'cacheManagerFactory' defined in file [F:\codebase\code\tour-tmsp\tmsp-server\target\classes\applicationContext-ehcache.xml]: Invocation of init method failed; nested exception is net.sf.ehcache.CacheException: Failed to parse EhCache configuration resource
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 34 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'cacheManagerFactory' defined in file [F:\codebase\code\tour-tmsp\tmsp-server\target\classes\applicationContext-ehcache.xml]: Invocation of init method failed; nested exception is net.sf.ehcache.CacheException: Failed to parse EhCache configuration resource
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1628)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 44 more
Caused by: net.sf.ehcache.CacheException: Failed to parse EhCache configuration resource
	at org.springframework.cache.ehcache.EhCacheManagerUtils.parseConfiguration(EhCacheManagerUtils.java:111)
	at org.springframework.cache.ehcache.EhCacheManagerFactoryBean.afterPropertiesSet(EhCacheManagerFactoryBean.java:135)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
	... 51 more
Caused by: java.io.FileNotFoundException: class path resource [ehcache.xml] cannot be opened because it does not exist
	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
	at org.springframework.cache.ehcache.EhCacheManagerUtils.parseConfiguration(EhCacheManagerUtils.java:107)
	... 54 more
2018-09-27 16:05:46 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:05:47 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:06:08 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:06:37 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:08:06 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :151(set)] Set keyex error : java.lang.NullPointerException
2018-09-27 16:08:07 [ ERROR ] [ RedisUtil.java :102(getJedis)] Get jedis error : redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
2018-09-27 16:13:25 [ ERROR ] [ DefaultSingletonBeanRegistry.java :581(destroyBean)] Destroy method on bean with name 'org.springframework.context.annotation.internalScheduledAnnotationProcessor' threw an exception
java.lang.IllegalStateException: ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: Root WebApplicationContext: startup date [Thu Sep 27 16:13:02 CST 2018]; root of context hierarchy
	at org.springframework.context.support.AbstractApplicationContext.getApplicationEventMulticaster(AbstractApplicationContext.java:404)
	at org.springframework.context.support.ApplicationListenerDetector.postProcessBeforeDestruction(ApplicationListenerDetector.java:97)
	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:253)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:578)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:554)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:523)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1033)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:555)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:800)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:444)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:294)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1349)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.startWebapp(JettyWebAppContext.java:296)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1342)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:741)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:505)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:365)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo.restartWebApp(JettyRunMojo.java:541)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo$1.filesChanged(JettyRunMojo.java:494)
	at org.eclipse.jetty.util.Scanner.reportBulkChanges(Scanner.java:685)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:539)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:391)
	at org.eclipse.jetty.util.Scanner$1.run(Scanner.java:329)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
2018-09-27 16:13:25 [ ERROR ] [ ContextLoader.java :350(initWebApplicationContext)] Context initialization failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'shiroFilter' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'securityManager' while setting bean property 'securityManager'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'securityManager' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'upmsRealm' while setting bean property 'realms' with key [0]; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is java.lang.RuntimeException: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.registerBeanPostProcessors(PostProcessorRegistrationDelegate.java:235)
	at org.springframework.context.support.AbstractApplicationContext.registerBeanPostProcessors(AbstractApplicationContext.java:702)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:527)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:443)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:325)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:800)
	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:444)
	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:791)
	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:294)
	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1349)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.startWebapp(JettyWebAppContext.java:296)
	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1342)
	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:741)
	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:505)
	at org.eclipse.jetty.maven.plugin.JettyWebAppContext.doStart(JettyWebAppContext.java:365)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:68)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo.restartWebApp(JettyRunMojo.java:541)
	at org.eclipse.jetty.maven.plugin.JettyRunMojo$1.filesChanged(JettyRunMojo.java:494)
	at org.eclipse.jetty.util.Scanner.reportBulkChanges(Scanner.java:685)
	at org.eclipse.jetty.util.Scanner.reportDifferences(Scanner.java:539)
	at org.eclipse.jetty.util.Scanner.scan(Scanner.java:391)
	at org.eclipse.jetty.util.Scanner$1.run(Scanner.java:329)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'securityManager' defined in URL [jar:file:/D:/dev/mavenlib/com/zheng/tmsp-client/1.0.0/tmsp-client-1.0.0.jar!/applicationContext-shiro.xml]: Cannot resolve reference to bean 'upmsRealm' while setting bean property 'realms' with key [0]; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is java.lang.RuntimeException: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveManagedList(BeanDefinitionValueResolver.java:382)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:157)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1531)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1276)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 34 more
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'upmsRealm': Unsatisfied dependency expressed through field 'upmsApiService'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is java.lang.RuntimeException: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:588)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:366)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1264)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 46 more
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'upmsApiService': FactoryBean threw exception on object creation; nested exception is java.lang.RuntimeException: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:175)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1634)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:254)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:208)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.addCandidateEntry(DefaultListableBeanFactory.java:1309)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1275)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1101)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1066)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:585)
	... 56 more
Caused by: java.lang.RuntimeException: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at javassist.CtClassType.getClassFile2(CtClassType.java:208)
	at javassist.compiler.MemberResolver.lookupMethod(MemberResolver.java:99)
	at javassist.compiler.MemberResolver.lookupMethod(MemberResolver.java:85)
	at javassist.compiler.TypeChecker.atMethodCallCore(TypeChecker.java:738)
	at javassist.compiler.TypeChecker.atCallExpr(TypeChecker.java:695)
	at javassist.compiler.JvstTypeChecker.atCallExpr(JvstTypeChecker.java:157)
	at javassist.compiler.ast.CallExpr.accept(CallExpr.java:46)
	at javassist.compiler.JvstTypeChecker.atCastToWrapper(JvstTypeChecker.java:126)
	at javassist.compiler.JvstTypeChecker.atCastExpr(JvstTypeChecker.java:98)
	at javassist.compiler.ast.CastExpr.accept(CastExpr.java:55)
	at javassist.compiler.CodeGen.doTypeCheck(CodeGen.java:242)
	at javassist.compiler.CodeGen.compileExpr(CodeGen.java:229)
	at javassist.compiler.CodeGen.atReturnStmnt2(CodeGen.java:615)
	at javassist.compiler.JvstCodeGen.atReturnStmnt(JvstCodeGen.java:425)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:363)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:351)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.CodeGen.atIfStmnt(CodeGen.java:398)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:355)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:351)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.MemberCodeGen.atTryStmnt(MemberCodeGen.java:204)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:367)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.CodeGen.atStmnt(CodeGen.java:351)
	at javassist.compiler.ast.Stmnt.accept(Stmnt.java:50)
	at javassist.compiler.CodeGen.atMethodBody(CodeGen.java:292)
	at javassist.compiler.CodeGen.atMethodDecl(CodeGen.java:274)
	at javassist.compiler.ast.MethodDecl.accept(MethodDecl.java:44)
	at javassist.compiler.Javac.compileMethod(Javac.java:169)
	at javassist.compiler.Javac.compile(Javac.java:95)
	at javassist.CtNewMethod.make(CtNewMethod.java:74)
	at javassist.CtNewMethod.make(CtNewMethod.java:45)
	at com.alibaba.dubbo.common.bytecode.ClassGenerator.toClass(ClassGenerator.java:282)
	at com.alibaba.dubbo.common.bytecode.ClassGenerator.toClass(ClassGenerator.java:257)
	at com.alibaba.dubbo.common.bytecode.Wrapper.makeWrapper(Wrapper.java:239)
	at com.alibaba.dubbo.common.bytecode.Wrapper.getWrapper(Wrapper.java:101)
	at com.alibaba.dubbo.config.ReferenceConfig.init(ReferenceConfig.java:290)
	at com.alibaba.dubbo.config.ReferenceConfig.get(ReferenceConfig.java:159)
	at com.alibaba.dubbo.config.spring.ReferenceBean.getObject(ReferenceBean.java:65)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:168)
	... 66 more
Caused by: javassist.NotFoundException: com.zheng.upms.rpc.api.UpmsApiService
	at javassist.CtClassType.getClassFile2(CtClassType.java:195)
	... 108 more
